extensions/agent/package.json

```diff
@@ -1,93 +1,93 @@
 {
-  "name": "agent",
-  "displayName": "SQL Server Agent",
-  "description": "Manage and troubleshoot SQL Server Agent jobs",
-  "version": "0.42.0",
-  "publisher": "Microsoft",
-  "preview": true,
-  "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt",
-  "icon": "images/sqlserver.png",
-  "aiKey": "AIF-5574968e-856d-40d2-af67-c89a14e76412",
-  "engines": {
-    "vscode": "^1.25.0"
-  },
-  "activationEvents": [
-    "*"
-  ],
-  "main": "./out/main",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Microsoft/azuredatastudio.git"
-  },
-  "extensionDependencies": [
-    "Microsoft.mssql"
-  ],
-  "contributes": {
-    "outputChannels": [
-      "sqlagent"
-    ],
-    "dashboard.tabs": [
-      {
-        "id": "data-management-agent",
-        "description": "Manage and troubleshoot SQL Agent jobs",
-        "provider": "MSSQL",
-        "title": "SQL Agent",
-        "when": "connectionProvider == 'MSSQL' && !mssql:iscloud",
-        "container": {
-          "controlhost-container": {
-            "type": "agent"
-          }
-        }
-      }
-    ],
-    "commands": [
-      {
-        "command": "agent.openNotebookDialog",
-        "title": "Schedule Notebook",
-        "icon": {
-          "dark": "resources/dark/open_notebook_inverse.svg",
-          "light": "resources/light/open_notebook.svg"
-        }
-      },
-      {
-        "command": "agent.reuploadTemplate",
-        "title": "Reupload Template",
-        "icon": {
-          "dark": "resources/dark/open_notebook_inverse.svg",
-          "light": "resources/light/open_notebook.svg"
-        }
-      }
-    ],
-    "menus": {
-      "notebook/toolbar": [
-        {
-          "command": "agent.openNotebookDialog",
-          "when": "providerId == sql"
-        },
-        {
-          "command": "agent.reuploadTemplate",
-          "when": "agent:trackedTemplate"
-        }
-      ],
-      "explorer/context": [
-        {
-          "command": "agent.openNotebookDialog",
-          "when": "resourceExtname == .ipynb"
-        }
-      ]
-    }
-  },
-  "dependencies": {
-    "vscode-nls": "^3.2.1"
-  },
-  "devDependencies": {
-    "mocha-junit-reporter": "^1.17.0",
-    "mocha-multi-reporters": "^1.1.7",
-    "@types/mocha": "^5.2.5",
-    "@types/node": "^8.10.25",
-    "mocha": "^5.2.0",
-    "should": "^13.2.1",
-    "typemoq": "^2.1.0",
-    "vscode": "1.1.5"
-  }
+	"name": "agent",
```

Comment:
It doesn't look like there's any changes in this file asides from spaces -> tabs? Please try to keep unrelated changes like this in separate PRs - it makes it more difficult to figure out what we should actually be reviewing.